composer.json

```diff
@@ -48,13 +49,17 @@
         "ibexa/design-engine": "~4.6.0@dev",
         "ibexa/code-style": "^1.0",
         "friendsofphp/php-cs-fixer": "^3.0",
-        "phpunit/phpunit": "^9.5"
+        "phpunit/phpunit": "^9.5",
+        "phpstan/phpstan": "^1.10",
```

Comment:
You can actually go and install v2 immediately. It's something we probably want to do with other packages too either.